packages/administration/templates/form/type/LocalizedType.html.twig

```diff
@@ -19,23 +19,51 @@
                 {{ form_help(form) }}
             </fieldset>
         {% else %}
-            {% for child in form %}
-                <div class="card mb-3">
-                    <div class="row">
-                        <div class="col-auto">
-                            <div class="card-body">
-                                {{ localeFlag(child.vars.attr['data-locale']) }}
+            {% if display_mode == 'columns' %}
+                {# Grid layout - 3 columns on desktop, full width on mobile #}
+                <div class="row">
+                    {% for child in form %}
+                        {# Determine the row class - priority: 1) parent child_row_class option, 2) entry_options row_attr, 3) default mb-3 #}
+                        {% set childRowClass = child_row_class|default(child.vars.row_attr.class|default('mb-3')) %}
+                        {% set childContent = form_row(child, {row_attr: {class: childRowClass}})|trim %}
+
+                        {% if childContent is not empty %}
+                            <div class="col-12 col-xl-4">
+                                <div class="card h-100">
+                                    <div class="card-header card-header-light py-2">
+                                        <span>
+                                            {{ localeFlag(child.vars.attr['data-locale']) }}
+                                        </span>
+                                    </div>
+                                    <div class="card-body">
+                                        {{- childContent|raw -}}
```

Comment:
<img width="1051" height="204" alt="Snímek obrazovky 2025-11-05 v 17 22 49" src="https://github.com/user-attachments/assets/ef179edc-0fe1-461a-ab90-3546627dfe56" />

It doesn't look nice when labels are visible here